QEfficient/base/modeling_qeff.py

```diff
@@ -132,7 +132,12 @@ def _export(
         """
         export_dir = Path(export_dir or (QEFF_HOME / self.model_name))
         export_dir = export_dir.with_name(export_dir.name + "-" + self.model_hash)
-        onnx_path = export_dir / f"{self.model_name}.onnx"
+        if self.num_speculative_tokens:
+            model_name = f"{self.model_name}_{self.num_speculative_tokens+1}nltk.onnx"
+        else:
+            model_name = f"{self.model_name}.onnx"
+        onnx_path = export_dir / model_name
+        # TODO: need to add hash to onnx
```

Comment:
the `model_hash` should be sufficient to differentiate between normal, TLM, DLM, TLM+CB etc  variations of model. Let's fix `model_hash` instead of doing this

Reply:
Did you check if this collision is happening because we consider only first 16 characters out of complete hash?